src/api/1/controller-collection/update-specifications/index.md

```diff
@@ -12,10 +12,69 @@ The updateSpecifications method allows you to create or update the validation sp
 
 When the validation specification is not formatted correctly, a detailed error message is returned to help you to debug.
 
+## Query Syntax
+
+{{{since "1.8.0"}}}
+
+### HTTP
+
+```http
+URL: http://kuzzle:7512/<index>/<collection>/_specifications
+Method: PUT  
+Body:
+```
+
+```js
+{
+  "strict": <boolean>,
+  "fields": {
+    // ... specification for each field
+  }
+}
+```
+
+### Other protocols
+
+```js
+{
+  "controller": "collection",
+  "action": "updateSpecifications",
+  "index": "myindex",
+  "collection": "mycollection",
+
+  "body": {
+    "strict": <boolean>,
+    "fields": {
+      // ... specification for each field
+    }
+  }
+  
```

Comment:
```suggestion
```

Reply:
I've found the previous snippet like this so, I think it's voluntary